github/users.go

```diff
@@ -144,6 +144,22 @@ type UserListOptions struct {
 
 // ListAll lists all GitHub users.
 //
+// To paginate through all users, populate 'Since' with the ID of the last user.
+// Example (warning: this will quickly eat through your GitHub API quota!):
```

Comment:
Ah!  I did not know that... but I see it is written right there... sorry I didn't read it through fully.
That is definitely a new factor.

OK, now comes the question: should the example go in `users_test.go` or do you have a preference for a different file naming convention for examples?

I can make a new PR that moves this example to where we think it should go.